### DISCUSSION TITLE
AspectJ Compile-Time Weaving and Transactional Method Proxies in Spring

### DISCUSSION THREAD

**Discussion by Sekator778** [dev-aspectj/aspectj-maven-plugin#149] [Q&A]
Will this plugin support compile-time weaving, allowing me to call transactional methods within the same class in a Spring application, and ensure that they are proxied and transactional behavior works correctly?

**Comment by kriegaex** [comment]
The answer is yes. 

Please read the documentation.

Sorry for my initial version of this answer, at first I thought you had created an issue, but it is a discussion item. As this is the first one of them, it came somewhat unexpectedly.
